modify import for ED visits

qing-duan opened the pull request:
modification only in import_clean_data.R

qing-duan committed:
modify import for ED visits

cole-brokamp committed:
Merge branch 'main' into EDVisits

cole-brokamp commented:
I added back in some commits that were on main that you didn't have pulled in before branching here.

cole-brokamp committed:
parse datetime on read

this won't require lubridate or any tidyr::separate, but
it still produces a parsing warning because some of the
datetime values are missing the time component; these
are parsed okay despite the warning with an as.Date() anyway

cole-brokamp committed:
update metadata field for new ED visit data

cole-brokamp committed:
update Makefile for new data input file

cole-brokamp committed:
change GUID column names

cole-brokamp committed:
update version of parcel to fix bug

related to failed parsing of a street name when trying to match
to known apartment street names

cole-brokamp committed:
multiprocess approach for AADT calculation

cole-brokamp committed:
use more recent version of traffic exposures

cole-brokamp committed:
update README

cole-brokamp committed:
v2.1.0
